jobs/src/test/scala/com/pennsieve/jobs/types/DeleteJobSpec.scala

```diff
@@ -279,14 +279,14 @@ class DeleteJobSpec
     assert(deleteJob.creditDeleteJob(msg).await.isRight)
```

Comment:
Note: I've made a ticket to come back and update these commented out tests
https://app.clickup.com/t/860pnxcq7